android/cli/commands/_build.js

```diff
@@ -3381,6 +3385,28 @@ AndroidBuilder.prototype.generateTheme = function generateTheme(next) {
 	next();
 };
 
+function serviceParser(serviceNode, result) {
```

Comment:
I don't like that you're passing in an object to modify as a side-effect of calling the method. I prefer to explicitly alter a copy of an object and return the modified copy as return value. Calling methods with the intention of causing side-effect modifications on the arguments is non-obvious.